### ISSUE TITLE
Missing `fontServerRevision` in AE 24.0

### ISSUE THREAD

**Issue by Klustre** [docsforadobe/after-effects-scripting-guide#31]
[FontsObject.fontServerRevision](https://ae-scripting.docsforadobe.dev/other/fontsobject.html#fontsobject-fontserverrevision) is available in beta 24.1 (Build 42) but not in production version 24.0 as expected.

cc @DouglasWaterfall @1Resonant

**Comment by DouglasWaterfall** [comment]
We have made no commitments in terms of specific production release dates for scripting features currently in beta.

Sometimes that is because we are still thinking about it, other times we are caught up doing other things - it is kinda fluid.

The APIs can change to.

**Comment by Klustre** [comment]
In that case the docs are incorrect. All the Font APIs have been added to 24.0, but for `fontServerRevision` (and others) the messaging is ambiguous:
```
This functionality was added in After Effects 24.0.
This functionality is currently in Beta and is subject to change.
```

**Comment by jthomascolombo** [comment]
[FontsObject.fontServerRevision](https://ae-scripting.docsforadobe.dev/other/fontsobject.html#fontsobject-fontserverrevision) is available as part of 24.2, indicated by updates made as part of dcd1258.
